### PR TITLE
register model at runtime

### PR DESCRIPTION
**Pull request by SzNagyMisu**
PR for #187 

90% done - stuck with remaining 10%...

I can register a brand new module into vuex (like `'entities/hobbies'`) but cannot add a new getter under an existing module (like the namespaced getter `'hobbies'` under the module `'entities'`).

See L44-L46 in `ModuleBuilder.createModules()`
https://github.com/vuex-orm/vuex-orm/blob/a48dea13b7e19351d415b7b9941d9bb934393927/src/modules/builder/Builder.ts#L35-L56

Any ideas?
@kiaking what are that gettters for? I don't remember having met them in the docs or during coding.

### CODE REVIEW

**Comment by kiaking** [comment]
Ahh... yeah. That getter is so we can do `store.getters['entities/users']()` and get Query instance of User model. It's shortcut for  `store.getters['entities/users/query']()`.

Looking into Vuex doc, maybe there's no way to add extra getters after store initialization... As you mentioned, it's not documented so maybe we can just remove that functionality.

Do you think you can remove the test for that getter and make this PR pass?

Thank you so much for the PR by the way! 🎉

File: test/feature/database/RegisterModel.spec.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        user: this.belongsTo(User, 'userId')
+      }
+    }
+  }
+
+  it('can register models before being installed to vuex.', async () => {
+    const database = new Database()
+
+    database.register(User)
+    database.register(Hobby) // TODO custom module?
```
Comment (SzNagyMisu) [inline test/feature/database/RegisterModel.spec.js]: @kiaking 
What do you think about testing this functionality with adding a custom module?

Comment (kiaking) [reply test/feature/database/RegisterModel.spec.js]: I think we should!

Comment (SzNagyMisu) [reply test/feature/database/RegisterModel.spec.js]: Added :smiley:

**Comment by kiaking** [comment]
@SzNagyMisu Hi sorry for my late response! Do you think you can resolve the conflict...!?

**Comment by SzNagyMisu** [comment]
Wow... Things have changed since... I'll give it a try

**Comment by SzNagyMisu** [comment]
I've decided it will be cleaner if I start the whole PR from scratch - so many things have changed since.

**Comment by SzNagyMisu** [comment]
Reopened PR at #535 - closing this one.

**Pull request closed without merging**